pkg/handlers/v1/books.go

```diff
@@ -2,56 +2,121 @@ package v1
 
 import (
 	"book_service/pkg/clients"
+	_const "book_service/pkg/constants"
 	"book_service/pkg/models/common"
 	"book_service/pkg/models/common/req"
+	"book_service/pkg/models/common/res"
 	"book_service/pkg/query"
 	"book_service/pkg/utils"
-	"fmt"
+	"net/http"
+
 	"github.com/gin-gonic/gin"
 	"github.com/google/uuid"
 	"github.com/jinzhu/copier"
 	"github.com/sirupsen/logrus"
-	"net/http"
 )
 
 var booksIndex, _ = utils.GetEnvVar[string]("BOOKS_INDEX", "books")
```

Comment:
Would move it the file that handles with Elastic.
Also, I wouldn't even send it in the handlers functions - it shouldn't know (or care) what is the specific implementation of the DAL